demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -17,28 +17,100 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
         registrar.addMethodCallDelegate(instance, channel: channel)
     }
     
+    struct qrCodeData{
+        static var requestURI = ""
+    }
+    
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-        if (call.method == "createDID"){
+        switch call.method {
+        case "createDID":
             createDid(result: result)
```

Comment:
Should probably be called `createDID` unless there's a convention against it

Reply:
(the all caps makes it easier to distinguish from the English word "did" - in some contexts it can look a bit confusing at a first glance)